full-customer-journey/src/app/signIn/page.tsx

```diff
@@ -38,7 +38,7 @@ const page = () => {
         </nav> */}
       </header>
       <section className="flex justify-center gap-x-8 items-center w-full">
-        <div className="w-[610px] flex flex-col items-center  border-2 border-red-300">
+        <div className="w-[610px] flex flex-col items-center ">
```

Comment:
Don't give fix width like this. instead use max-w-screen-2xl or whatever you required, and then use padding. This is how it will be responsive not fixed.